test/framework/tinkerbell.go

```diff
@@ -62,13 +63,20 @@ func NewTinkerbell(t *testing.T, opts ...TinkerbellOpt) *Tinkerbell {
 		},
 	}
 
+	tink.cidr = os.Getenv(tinkerbellNetworkCidrEnvVar)
```

Comment:
Not something to be addressed here as I've seen it throughout our code base but we want to avoid arbitrarily reading the environment, including in tests. 

Mutating and retrieving the env mid program can result in accidental behavior and make the code inflexible. Configuration is a concern in its own right and should be handled as such.